static/script/devices/mediaplayer/samsung_streaming_2015.js

```diff
@@ -0,0 +1,115 @@
+/**
+ * @preserve Copyright (c) 2017-present British Broadcasting Corporation. All rights reserved.
+ * @license See https://github.com/fmtvp/tal/blob/master/LICENSE for full licence
+ */
+
+require.def(
+    'antie/devices/mediaplayer/samsung_streaming_2015',
+    [
+        'antie/devices/device',
+        'antie/devices/mediaplayer/mediaplayer',
+        'antie/devices/mediaplayer/samsung_streaming',
+        'antie/runtimecontext'
+    ],
+    function(Device, MediaPlayer, SamsungStreaming, RuntimeContext) {
+        'use strict';
+
+        /**
+         * MediaPlayer implementation for Samsung 2015 devices supporting HLS Live Seek implementing the Streaming API.
+         * Use this device modifier if a device implements the Samsung Streaming media playback standard.
+         * It must support creation of &lt;object&gt; elements with appropriate SAMSUNG_INFOLINK classids.
+         * Those objects must expose an API in accordance with the Samsung Streaming media specification.
+         * @name antie.devices.mediaplayer.SamsungStreaming2015
+         * @class
+         * @extends antie.devices.mediaplayer.SamsungStreaming
+         */
+        var Player = SamsungStreaming.extend({
+            init: function() {
+                this._super();
```

Comment:
I believe this function can be deleted as it appears to be identical to `init()` in the superclass (https://github.com/bbc/tal/blob/master/static/script/devices/mediaplayer/samsung_streaming.js#L53).

Subclasses should only implement their own `init()` if they need to behave differently to their superclass, and even then any common functionality should be handled by calling `this._super()`.